doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -430,6 +430,12 @@ Bluetooth Mesh samples
 Cellular samples
 ----------------
 
+* Added support for the nRF9151 DK in all cellular samples except for the following samples:
```

Comment:
Updated the change log as support was added as part of PR https://github.com/nrfconnect/sdk-nrf/pull/14161